locales/en/index.yml

```diff
@@ -127,6 +127,16 @@ inbox:
   enableCallToActionDescription: Enable the inbox to view messages
 profile:
   fiscalCode:
+    accessibility:
+      preview:
+        label: Preview of your fiscal code document
+        hint: Activate to show the facsimile of your fiscal code document
+      front:
+        label: Front-side of the document code {{code}} of {{name}} {{family_name}}
```

Comment:
Could we add all other informations?
- place of birth
- sex
- province
- date of birth

Reply:
added in ca541de